# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ipc/PAPZ.ipdl
# Commit: 76279f0a6d33
# Full Hash: 76279f0a6d33d6fe128a9f3aabe5393e52181818
# Author: James Willcox <snorp@snorp.net>
# Date: 2020-06-08 21:38:11
# Regressor Bug: 1536833
# File Overlap Count: 1
# Description:
#   Bug 1536833 - Fix overscroll effect on Android r=botond,geckoview-reviewers,agi
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78130
# ==============================================================================

diff -r ad757ab87752 -r 76279f0a6d33 gfx/layers/ipc/PAPZ.ipdl
--- a/gfx/layers/ipc/PAPZ.ipdl	Sun Jun 07 23:15:45 2020 +0000
+++ b/gfx/layers/ipc/PAPZ.ipdl	Mon Jun 08 17:13:28 2020 +0000
@@ -54,9 +54,9 @@
 
   async RequestContentRepaint(RepaintRequest request);
 
-  async UpdateOverscrollVelocity(float aX, float aY, bool aIsRootContent);
+  async UpdateOverscrollVelocity(ScrollableLayerGuid aGuid, float aX, float aY, bool aIsRootContent);
 
-  async UpdateOverscrollOffset(float aX, float aY, bool aIsRootContent);
+  async UpdateOverscrollOffset(ScrollableLayerGuid aGuid, float aX, float aY, bool aIsRootContent);
 
   async NotifyMozMouseScrollEvent(ViewID aScrollId, nsString aEvent);
 